src/excmds.ts

```diff
@@ -1706,11 +1706,65 @@ export async function tabprev(increment = 1) {
     // Proper way:
     // return tabIndexSetActive((await activeTab()).index - increment + 1)
     // Kludge until https://bugzilla.mozilla.org/show_bug.cgi?id=1504775 is fixed:
-    return browser.tabs.query({ currentWindow: true, hidden: false }).then(tabs => {
-        tabs = tabs.sort((t1, t2) => t1.index - t2.index)
-        let prevTab = (tabs.findIndex(t => t.active) - increment + tabs.length) % tabs.length
-        return browser.tabs.update(tabs[prevTab].id, { active: true })
-    })
+
+    if (config.get("treestyletabintegration")) {
```

Comment:
Can this go in a separate file? I'd rather have `if config.get... interop.tst.tab(...)` etc so if TST changes its API we only have to look at one file

Reply:
Yeah, probably best to shove it in it's own file, wasn't quite sure of the structure of the whole project so I initially just shoehorned it into one existing function.

Reply:
Where should I throw the file? Is there an existing structure that I should take advantage of?